jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/TryPathsHandlerTest.java

```diff
@@ -94,7 +96,16 @@ public void dispose()
     @Test
     public void testTryPaths() throws Exception
     {
-        ResourceHandler resourceHandler = new ResourceHandler();
+        ResourceHandler resourceHandler = new ResourceHandler()
+        {
+            @Override
+            protected HttpContent.Factory setupContentFactory()
+            {
+                // We don't want to cache not found entries for this test.
```

Comment:
There should be a more straightforward way to achieve this. Perhaps a config setting?

Reply:
I think there would be too many settings to introduce config for.
And this doesn't have init parameters like the `DefaultServlet`.

Reply:
I would have expected `ResourceHandler` to have a `setHttpContentFactory(HttpContent.Factory)` so that here I could have set just the `ResourceHCF`.